packages/boot/src/__tests__/acceptance/application-metadata.booter.acceptance.ts

```diff
@@ -10,7 +10,7 @@ import {BooterApp} from '../fixtures/application';
 
 describe('application metadata booter acceptance tests', () => {
   let app: BooterApp;
-  const sandbox = new TestSandbox(resolve(__dirname, '../../.sandbox'));
+  const sandbox = new TestSandbox(resolve(__dirname, '../../.sandbox'), '.');
```

Comment:
Can we please find a way how to use a unique subdirectory in this test too? What is preventing us from doing that now?

Reply:
I explained it in the code comment.